src/urllib3/connectionpool.py

```diff
@@ -220,11 +220,6 @@ def __init__(
         self.conn_kw = conn_kw
 
         if self.proxy:
-            # Enable Nagle's algorithm for proxies, to avoid packet fragmentation.
```

Comment:
This comment would indicate that the intent here was to do something different than what the code did. I suspect something got lost before we had sufficient code coverage

Reply:
Possibly author just forgot to make the correct/final implementation.

Reply:
This is the bug location, the author used the 'user provided defaults" to delete the options, instead of implementing another mechanism. I just realized that I bugged now the `HTTPConnectionPool` (dizzy last night), I'll go on to see to provide a final fix that keeps `HTTPConnectionPool` intact.

Reply:
=> not actionable